### ISSUE TITLE
Invalid URI. Load of media resource  failed.

### ISSUE THREAD

**Issue by rancor1223** [Chirmaya/BooruSlideshow#42]
I'm having trouble loading with "danbooru.donmai.us" when searching for "pool:10374". 

This used to work just fine couple months back, but now it only loads 7 images and the rest is greyed out. When I click the greyed out images, the console reads "Invalid URI. Load of media resource  failed.". I'm assuming the website changed their API or something?

**Comment by Chirmaya** [comment]
Thanks for the report! This should now be fixed in the last release (v7.17)

**Comment by FengLengshun** [comment]
Not sure if this is the same problem, but from time to time, I seem to have this problem where it's only spinning BS logo but squished into very small space. None of the rest would load. I checked on Chrome and it had that problem for a while, but just as I was typing this report, it got fixed (along with the "Error Loading [site]" message it kept showing at the top of the page).

Seems like it updated to v7.17 in the background while Firefox is still on v1.14?
